spec/models/contributor_spec.rb

```diff
@@ -360,7 +360,7 @@
 
     describe 'number of database calls' do
       subject { -> { contributor.recent_replies.first.request } }
-      it { should make_database_queries(count: 1) }
+      it { should make_database_queries(count: 2) }
```

Comment:
Why :thinking: ?

Reply:
Just for reference, this is the solution recommended in the [Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations) to load associated records. It requires an additional database query per association (i.e. the total numbers of queries is still constant), but requires less memory.